src/style.css

```diff
@@ -1 +1,62 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
Don't add anything in * selector except box-sizing: border-box, checklist p.9
![image](https://user-images.githubusercontent.com/118751034/211880105-27413148-1b73-40b4-b78a-9253cdbe0da1.png)